<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QTextEdit" name="txt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QScrollBar" name="verticalScrollBar">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>utenti</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionindietro"/>
   <addaction name="actionlogo"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionundo"/>
   <addaction name="actionredo"/>
   <addaction name="separator"/>
   <addaction name="actiontaglia"/>
   <addaction name="actioncopia"/>
   <addaction name="actionincolla"/>
   <addaction name="separator"/>
   <addaction name="actionallineaS"/>
   <addaction name="actionallineaC"/>
   <addaction name="actionallineaD"/>
   <addaction name="actionalineaG"/>
   <addaction name="separator"/>
   <addaction name="actiongrassetto"/>
   <addaction name="actioncorsivo"/>
   <addaction name="actionsottolineato"/>
  </widget>
  <action name="actionindietro">
   <property name="text">
    <string>indietro</string>
   </property>
  </action>
  <action name="actionlogo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/img/logo.png</normaloff>
     <disabledoff>:/images/img/logo.png</disabledoff>
     <disabledon>:/images/img/logo.png</disabledon>:/images/img/logo.png</iconset>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </action>
  <action name="actionlogoU">
   <property name="text">
    <string>logoU</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="text">
    <string>redo</string>
   </property>
  </action>
  <action name="actioncopia">
   <property name="text">
    <string>copia</string>
   </property>
  </action>
  <action name="actionincolla">
   <property name="text">
    <string>incolla</string>
   </property>
  </action>
  <action name="actiontaglia">
   <property name="text">
    <string>taglia</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actionallineaD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>allineaD</string>
   </property>
  </action>
  <action name="actionallineaS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>allineaS</string>
   </property>
  </action>
  <action name="actionallineaC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>allineaC</string>
   </property>
  </action>
  <action name="actionalineaG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>alineaG</string>
   </property>
  </action>
  <action name="actiongrassetto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>grassetto</string>
   </property>
  </action>
  <action name="actioncorsivo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>corsivo</string>
   </property>
  </action>
  <action name="actionsottolineato">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>sottolineato</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
